.travis.yml

```diff
@@ -1,72 +0,0 @@
----
```

Comment:
Lets land the new thing and delete this file once we have confirmed it is working (after landing to master and having a few passing CI runs).

Reply:
Added it back for now. Seems reasonable.